src/pattern-library/examples/non-popover-listbox.tsx

```diff
@@ -0,0 +1,70 @@
+import { useId, useState } from 'preact/hooks';
+
+import SelectNext from '../../components/input/SelectNext';
```

Comment:
The import here uses an internal path, but it would be nicer if written using the same syntax as a consumer of the library would write.

Since the import path is already documented on the page, we could also just save some space when the source is rendered by stripping out imports.

Reply:
Right! I thought about that, as it would be nicer to have `import { SelectNext } from '@hypothesis/frontend-shared';`.

I also thought about stripping imports as a simple workaround. I think I'll try that for now.